client/src/components/refreshrsPage/Refreshr.js

```diff
@@ -78,26 +72,47 @@ function Refreshr(props) {
     };
     const data = {
       title: questionObject.refreshrName,
+      variables: {
+        score: 0
+      },
       fields: [
         {
-          title: questionObject.reviewText,
+          title: 'Please enter your email address.',
+          type: 'email',
+          validations: {
+            required: true
+          }
+        },
+        {
+          ref: 'question_1',
+          title: questionObject.questionTextOne,
           type: 'multiple_choice',
           properties: {
+            randomize: true,
```

Comment:
Assuming this randomizes the order of the answers below?

Reply:
Yep - exactly. It randomizes all the questions the user will see